<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Get_result">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>390</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Расчет</string>
    </property>
   </widget>
   <widget class="QListWidget" name="Kriterii">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>371</width>
      <height>191</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Rang | assigment | function Min/Max</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>________________________________________________________________________________________________________</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="Ogran">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Function        |      Equal type            |                 num</string>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>_______________________________________________________________________________________________________</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="C_function_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Уравнение критерия</string>
    </property>
   </widget>
   <widget class="QLabel" name="C_Validate_label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>330</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Важность критерия</string>
    </property>
   </widget>
   <widget class="QLabel" name="C_Min_max_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>330</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Min/Max</string>
    </property>
   </widget>
   <widget class="QLabel" name="C_Assigment_label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>330</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Уступка в процентах 0.ХХХ....</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="C_function_edit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5x1+2x2+3x3-4x4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="C_Min_Max_edit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>max</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="C_Validate_edit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>360</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="C_Assigment_edit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>360</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0.25</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>410</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QLabel" name="Yravn_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Критерии</string>
    </property>
   </widget>
   <widget class="QLabel" name="Yravn_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ограничения</string>
    </property>
   </widget>
   <widget class="QLabel" name="Yravn_4">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Change_Krit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Change_Ogran">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить ограничение</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить критерий</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="O_Equal_val">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>360</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="O_function_edit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5x14x4</string>
    </property>
   </widget>
   <widget class="QLabel" name="O_function_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Уравнение ограничение</string>
    </property>
   </widget>
   <widget class="QLabel" name="O_Equal_num_label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>330</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Равенство</string>
    </property>
   </widget>
   <widget class="QComboBox" name="O_Eque_Sym_Combobox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>&gt;=</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt;=</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="O_Equal_t_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>330</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Equal type</string>
    </property>
   </widget>
   <widget class="QListWidget" name="Result">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>191</width>
      <height>341</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Symbol       =     Value</string>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>_______________________________________________________________________________________________________</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>935</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
